types/types.go

```diff
@@ -61,7 +61,7 @@ func ReadStats(path string) FuzzerStats {
 
 // Read the contents of a directory out
 func ReadDir(path string) InputCorpus {
-	corpus := InputCorpus{}
+	corpus := InputCorpus{[]Input{}}
```

Comment:
Needed so that loading a corpus with no `Input`s doesn't end up with a null pointer error.